de.metas.adempiere.adempiere/base/src/main/java-legacy/org/adempiere/model/RelationTypeZoomProvider.java

```diff
@@ -220,7 +246,23 @@ private MQuery mkQuery(final IZoomSource zoomSource, final ZoomProviderDestinati
 		final StringBuilder queryWhereClause = new StringBuilder();
 		final ITableRefInfo refTable = target.getTableRefInfo();
 		final String refTableWhereClause = refTable.getWhereClause();
-		if (!Check.isEmpty(refTableWhereClause))
+
+		if (target.isReferenceTarget)
+		{
+			queryWhereClause
```

Comment:
AFAIR you once implemented support for multiple and somehow prefixed `AD_Table_ID/Record_ID` pairs.
Please also let this zoom-to feature support them

Reply:
I am sorry but I don't remember anything like this. Can you, please, provide more info, if you remember?